### DISCUSSION TITLE
GdUnit4 3D Platformer tutorial video (scene_runner)

### DISCUSSION THREAD

**Discussion by ACG8** [MikeSchulze/gdUnit4#268] [Show and tell]
Hello again @MikeSchulze , just wanted to let you know that I have made another GdUnit4 tutorial:

https://www.youtube.com/watch?v=m6tYigD6Oe0

This one focuses on using scene_runner. If you like it, please feel free to link to it on the wiki.

**Comment by MikeSchulze** [comment]
Thank you, very good, I left a comment on the tutorial.
I will definitely include this tutorial in the documentation.

Thanks again, keep up the good work ;)

**Comment by MikeSchulze** [comment]
@ACG8  You are also welcome to post it on [Godot Discord Tutorials ](https://discord.com/channels/212250894228652034/319225525052899328)

**Reply by ACG8** [reply]
Thanks! I don't think I have access to that channel, though (this is on the Godot Engine discord, right?); do you know how I get access to that channel?

**Reply by ACG8** [reply]
Never mind, I think I figured it out.

**Comment by JonBoyleCoding** [comment]
Great video! This is precisely what I needed to learn how to use the scene runner (I was debating between GUT and gdUnit4 until I saw through this video!).

**Reply by ACG8** [reply]
I'm so glad you found it helpful!
